.github/workflows/build_and_test_rolling.yaml

```diff
@@ -0,0 +1,76 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: Build and Test (rolling)
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  industrial_ci:
+    strategy:
+      matrix:
+        env:
+          - ROS_DISTRO: rolling
+            ROS_REPO: main
+          - ROS_DISTRO: rolling
+            ROS_REPO: testing
+          # galactic is not yet supported by industrial_ci
+          # - ROS_DISTRO: galactic
```

Comment:
I just tried galactic and it seems to work just as well with the provided workspace dependencies. There are only some deprecation and formatting warnings that we still need to fix. Any reason not to enable this as well?